charts/seed-bootstrap/values.yaml

```diff
@@ -111,6 +111,15 @@ allowedMetrics:
   - fluentbit_filter_add_records_total
   - fluentbit_filter_drop_records_total
 
+  fluentbitOutputPlugin:
+  - promtail_dropped_entries_total
+  - output_plugin_errors
+  - output_plugin_no_metadata
+  - output_plugin_incoming_logs
+  - output_plugin_past_send_requests
+  - output_plugin_dropt_logs
```

Comment:
I think it might be a bit more involved to fix this typo: `dropt`, as it is coming from the fluent-bit-to-loki plugin,
https://github.com/gardener/logging/blob/99b64c43b0a2e84e833b74d318cad8292431ad34/pkg/metrics/metrics.go#L54
but better now then later :) [drop -> dropped](https://www.oxfordlearnersdictionaries.com/definition/english/drop_1)

Reply:
Yes, I know already about this typo, but as you mentioned, it is not coming from this PR.

Reply:
So lets hold this PR until I release the logging repo